app/views/formio/submissions/edit.html.haml

```diff
@@ -0,0 +1,11 @@
+:javascript
+  window.onload = function() {
+    Formio
+      .createForm(document.getElementById('formio'), '#{@formio_url}')
+      .then(function(form) {
+        form.on('submitDone', function(submission) {
+          var surveyUrl = '#{@formio_url}';
```

Comment:
I notice that you use `encodeURIComponent` below and not here; just wanted to make sure that was intentional.

Reply:
Good call! In this case, things work because I’m not munging `@formio_url`, but I’ll change for consistency and to avoid this question in the future.